README.md

```diff
@@ -53,7 +59,8 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+the declared variable weather is the string 'sunny' as shown by the single equals sign.
+in the second expression weather is "sunny" as shown by the triple equals sign known as a strict comparision operator.
```

Comment:
Sort of. The third expression is a comparison operation - it will return either `true` or `false` based on whether or not the two arguments are equal.